lib/safeRequire.js

```diff
@@ -3,7 +3,9 @@ module.exports = function safeRequire (library, feature) {
 		return require(library);
 	} catch (error) {
 		if (error.code === 'MODULE_NOT_FOUND') {
-			throw new Error('\nTo use ' + feature + ' install ' + library);
+			console.error('\nTo use ' + feature + ' install ' + library);
+			process.exit(1);
```

Comment:
I switched to using `process.exit(1)` because most of these cases were already doing that and it seems to make more sense.